master/buildbot/www/hooks/github.py

```diff
@@ -128,21 +136,30 @@ def handle_push(self, payload, event):
 
         return changes, 'git'
 
+    @defer.inlineCallbacks
     def handle_pull_request(self, payload, event):
         changes = []
         number = payload['number']
         refname = 'refs/pull/{}/merge'.format(number)
         commits = payload['pull_request']['commits']
         title = payload['pull_request']['title']
         comments = payload['pull_request']['body']
+        repo_url = payload['repository']['url']
+        head_sha = payload['pull_request']['head']['sha']
+
+        head_msg = yield self._get_commit_msg(repo_url, head_sha)
+        log.msg("head commit message {}".format(head_msg))
+        log.msg("head commit message has skip {}".format(self._has_skip(head_msg)))
+        if self._has_skip(head_msg):
+            defer.returnValue((changes, 'git'))
```

Comment:
I would rather say

    defer.returnValue(([], 'git'))

to show that we mean to return no change